app/assets/stylesheets/content/_tabs.sass

```diff
@@ -69,7 +69,7 @@
     pointer-events: none
     border-bottom-width: 0
     a
-      color: $tabs-font-color-disabled
+      color: tabs-font-color-disabled
```

Comment:
Please revert this change. Without the `$`it is not recognised as a variable and simply ignored.